src/org/usfirst/frc/team2175/command/single/DriveInchesWithPercentVbusCommand.java

```diff
@@ -0,0 +1,43 @@
+package org.usfirst.frc.team2175.command.single;
+
+import org.usfirst.frc.team2175.ServiceLocator;
+import org.usfirst.frc.team2175.command.BaseCommand;
+import org.usfirst.frc.team2175.subsystem.drivetrain.DrivetrainSubsystem;
+import org.usfirst.frc.team2175.subsystem.drivetrain.TrapezoidalMotionProfile;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class DriveInchesWithPercentVbusCommand extends BaseCommand {
```

Comment:
@NoahConnors Is this class ok to use in our code? If not, I would recommend putting `@Deprecated` above the class (right before line 10 here) so that Eclipse can help us remember not to use it at competition.